website/docs/docs/core/connect-data-platform/redshift-setup.md

```diff
@@ -38,7 +38,8 @@ import SetUpPages from '/snippets/_setup-pages-intro.md';
 | `db_groups`  | ['ANALYSTS'] | Optional. A list of existing database group names that the DbUser joins for the current session |
 | `ra3_node`  | true | Optional, default False. Enables cross-database sources|
 | `autocommit`  | true | Optional, default True. Enables autocommit after each statement|
-| `retries`  | 1 | Number of retries |
+| `retries`  | 1 | Number of retries (on each statement) |
+| `retry_all`  | true | Allows dbt to retry all statements in a query |
```

Comment:
```suggestion
| `retry_all`  | true | Allows dbt to retry all statements in a query* |
```

*We recommend configuring this to address intermittent Redshift Driver errors like `could not open relation with OID XXXXX`.